submissions/damisiMahmood/html-forms/Contact Page.html

```diff
@@ -0,0 +1,61 @@
+<!Doctype html>
```

Comment:
Remove spaces in this file name

Reply:
Done

Reply:
Adaa, the images are a part of my website, they are in the header and footer of my website.

Kindly note